pkg/controller/cloneset/sync/cloneset_sync_utils.go

```diff
@@ -241,3 +243,7 @@ func isPodAvailable(coreControl clonesetcore.Control, pod *v1.Pod, minReadySecon
 	}
 	return coreControl.IsPodUpdateReady(pod, minReadySeconds)
 }
+
+func getReadinessMessage(cs *appsv1alpha1.CloneSet) podreadiness.Message {
+	return podreadiness.Message{UserAgent: "CloneSet", Key: fmt.Sprintf("%s/%s", cs.Namespace, cs.Name)}
```

Comment:
The Message must be unique. So maybe `podreadiness.Message{UserAgent: "CloneSet", Key: "PreDeleteHook"}`

(This is different with the CRR. There might be multiple CRRs for one Pod, so we have to put the name of CRR in key to declare which one triggers not-ready. But one Pod only belongs to a CloneSet, so we don't have to put the name of CloneSet in it)